src/v2/Apps/RSS/templates/article.ejs

```diff
@@ -0,0 +1,60 @@
+<item>
+  <guid isPermaLink="false"><%= appUrl %><%= article.href %></guid>
+  <title><%= article.thumbnailTitle %></title>
+  <link><%= appUrl %><%= article.href %></link>
+  <author><%= article.byline %></author>
```

Comment:
This is technically Invalid FWIW. This is the old behavior but according to spec `author` should be the author's email address. I'm assuming other RSS feeds abuse this field too.